bool enable
---
bool was_enabled
